deployment/helm/skaha/init-users-groups-config/init-users-groups-from-service.sh

```diff
@@ -84,11 +84,11 @@ else
 
     # For the case of using the CADC client certificate.
     if [[ -f "${CADC_PROXY_CERT_FILE}" ]] ; then
-        curl -SsL -E ${CADC_PROXY_CERT_FILE} "${UID_URL}" | sed "s/^\([a-zA-Z0-9_\@\.\-]*\):\(.*\):::$/\1:\2::${ESCAPED_HOME}\/\1:\/sbin\/nologin/" >> "${PASSWD_FILE}"
-        curl -SsL -E ${CADC_PROXY_CERT_FILE} "${GID_URL}" >> "${GROUP_FILE}"
+        curl -kSsL -E ${CADC_PROXY_CERT_FILE} "${UID_URL}" | sed "s/^\([a-zA-Z0-9_\@\.\-]*\):\(.*\):::$/\1:\2::${ESCAPED_HOME}\/\1:\/sbin\/nologin/" >> "${PASSWD_FILE}"
```

Comment:
Because injecting the CA Certificate for self-signed certificates is more work for this `initContainer`, unfortunately.  So for my testing on the `cadc.dao.nrc.ca` domain, this is necessary.  Once #666 is merged, this will all go away.